.github/workflows/codeql.yml

```diff
@@ -55,18 +55,15 @@ jobs:
 
     # Autobuild attempts to build any compiled languages  (C/C++, C#, Go, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v2
 
     # ℹ️ Command-line programs to run using the OS shell.
     # 📚 See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
 
     #   If the Autobuild fails above, remove it and uncomment the following three lines.
     #   modify them (or add more) to build your code if your project, please refer to the EXAMPLE below for guidance.
 
-    # - run: |
-    #   echo "Run, Build Application using script"
-    #   ./location_of_script_within_repo/buildscript.sh
+    - run: |
```

Comment:
Do not clear why Makefile is deleted, could you add comment/name to clarifying?

Reply:
Makefile is deleted in order to skip triggering make build by codeanalyze which then fails with downloaded ./go

Reply:
Thats the reason why we removed, i think to be clear better explicitly say about it and add name for that step.

Reply:
Truth be told we may mv it to Makefile.backup and after codeql step return it back